featuretests/package_test.go

```diff
@@ -27,6 +27,9 @@ func init() {
 		panic(err)
 	}
 
+	// TODO (go1.13)
+	// This will fail when we go to go 1.13
+	// https://github.com/golang/go/issues/33774
 	flag.Parse()
```

Comment:
```
	if match, _ := regexp.Match("go1.1[0-2](^\\d)*", []byte(runtime.Version())); match {
		flag.Parse()
	}
```

Reply:
Well, I do not think that this will help us as we'd still want to parse the flags even for go 1.13+.
May be the parsing will have to be done at suite setup... Have not given this much thought since we are not ready for go 1.13 yet... and I have since reverted my local go version to 1.12...